src/sass/main.scss

```diff
@@ -5,10 +5,25 @@ html {
 }
 
 // used for theme skins with a default to gray
-@mixin color_cascade($color: #999) {
+@mixin color_cascade($color: #999, $alert-color: red, $warning-color: orange) {
```

Comment:
Hm, actually this is totally fine. I was thinking it was doing something differently, but then forgot about the whole "skins" concept. Will clean up some things but otherwise the CSS is good to go.